ports/stm/mpconfigport.mk

```diff
@@ -41,7 +41,7 @@ ifeq ($(MCU_SERIES),F7)
 	CIRCUITPY_ANALOGIO = 0
 	CIRCUITPY_NEOPIXEL_WRITE = 0
 	CIRCUITPY_PULSEIO = 0
-	CIRCUITPY_OS = 0
+	CIRCUITPY_OS = 1
```

Comment:
Please also enable for the H7, this change applies to both.

Reply:
I can't test on a H7, and I don't like to enable things without testing them.